### PR TITLE
[7.13.x-blue] [KIECLOUD-647] Remove OpenShift templates from rhpam-7-openshift-image

### PR DESCRIPTION
**Pull request by davidesalerno**
Signed-off-by: Davide Salerno <dsalerno@redhat.com>

See: [KIECLOUD-647](https://issues.redhat.com/browse/KIECLOUD-647)

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,10 +28,9 @@ Inside each image directory you will find the following files:
  - **image.yaml**: the main image descriptor file, here are all the pieces and configuration needed to build an image.
 
 
-On the root directory you can also find the two files:
+On the root directory you can also one additiona file:
```
Comment (spolti) [inline README.md:31]:
```suggestion
On the root directory you can also find one additional file:
```

File: quickstarts/hello-rules-multi-module/README.md [hunk 2]
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
...
-```
-
-Default credential is `adminUser/RedHat`.
-
 Note that, to pull the images the OpenShift must be able to pull images from registry.redhat.io, for more information
 please take a look [here](https://access.redhat.com/RegistryAuthentication)
 
-At this moment we are ready to instantiate the kieserver app:
+To create the IBM BAMOE environment on your OpenShift instance you need to install Business Automation Operator.
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```
Comment (spolti) [inline quickstarts/hello-rules-multi-module/README.md:28]:
```suggestion
For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```

File: quickstarts/hello-rules-multi-module/README.md [hunk 3]
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
...
 
+Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```
Comment (spolti) [inline quickstarts/hello-rules-multi-module/README.md:30]:
```suggestion
Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```

File: quickstarts/hello-rules-multi-module/README.md [hunk 4]
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
...
-$ oc new-app rhdm713-prod-immutable-kieserver \
--p KIE_SERVER_HTTPS_SECRET=decisioncentral-app-secret \
--p CREDENTIALS_SECRET=rhpam-credentials \
--p KIE_SERVER_CONTAINER_DEPLOYMENT=hellorules=org.openshift.quickstarts:rhpam-kieserver-decisions:1.6.0-SNAPSHOT \
--p ARTIFACT_DIR=hellorules/target,hellorules-model/target \
--p SOURCE_REPOSITORY_URL=https://github.com/jboss-container-images/rhpam-7-openshift-image.git \
--p SOURCE_REPOSITORY_REF=main \
--p CONTEXT_DIR=quickstarts/hello-rules-multi-module \
--p IMAGE_STREAM_NAMESPACE=openshift
+If you view the generated YAML source for the installation, the source should be similar to the following example:
```
Comment (spolti) [inline quickstarts/hello-rules-multi-module/README.md:32]:
```suggestion
If you view the generated YAML source for the installation, the source should be similar to the following example:

```

File: quickstarts/hello-rules-multi-module/README.md [hunk 5]
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
...
+metadata:
+   name: hello-rules-quickstart
+spec:
+   environment: rhpam-production-immutable
+...
 ```
 
-
-Now you can deploy the [hellorules-client](hellorules-client) in the same or another project and test IBM BAMOE KIE Server container.
+When your KieApp will be ready and running, you can go ahead deploying  the [hellorules-client](hellorules-client) in the same or another project and test IBM BAMOE KIE Server container.
```
Comment (spolti) [inline quickstarts/hello-rules-multi-module/README.md:43]:
```suggestion
When the KieApp gets ready, you can go ahead deploying  the [hellorules-client](hellorules-client) in the same or another project and test IBM BAMOE KIE Server container.
```

File: quickstarts/hello-rules/README.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
 ## IBM Business Automation Manager Open Editions KIE Server decisions Quickstart
 
-Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/quickstarts/hello-rules)
+Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x/quickstarts/hello-rules)
```
Comment (spolti) [inline quickstarts/hello-rules/README.md:3]:
```suggestion
Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/quickstarts/hello-rules)
```

File: quickstarts/hello-rules/README.md [hunk 7]
```diff
@@ -20,67 +20,30 @@ $ oc login https://<your_openshift_address>:<port>
...
-
-If the `rhpam-kieserver-rhel8` is not found, install it under the 'openshift' namespace:
-```bash
-$ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/main/rhdm713-image-streams.yaml -n openshift
-```
 Note that, to pull the images the OpenShift must be able to pull images from registry.redhat.io, for more information
 please take a look [here](https://access.redhat.com/RegistryAuthentication)
 
+To create the IBM BAMOE environment  on your OpenShift instance you need to install Business Automation Operator.
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```
Comment (spolti) [inline quickstarts/hello-rules/README.md:31]:
```suggestion
For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```

File: quickstarts/hello-rules/README.md [hunk 8]
```diff
@@ -20,67 +20,30 @@ $ oc login https://<your_openshift_address>:<port>
...
-
-```bash
-$ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/main/example-credentials.yaml
-secret/rhpam-credentials created
-```
-
-Default credential is `adminUser/RedHat`.
-
-At this moment we are ready to instantiate the kieserver app:
+Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```
Comment (spolti) [inline quickstarts/hello-rules/README.md:33]:
```suggestion
Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```

File: quickstarts/hello-rules/README.md [hunk 9]
```diff
@@ -125,35 +88,9 @@ by using the *hello-rules* quickstart and this client to interact with Active MQ
...
- - install the https secret
- - credentials secret
- - Make sure that the Active MQ 7.8 imagestream is available in the `openshift` namespace. 
-   ```bash
-   $ oc replace -n openshift --force  -f \
-     https://raw.githubusercontent.com/jboss-container-images/jboss-amq-7-broker-openshift-image/78-7.8.0.GA/amq-broker-7-image-streams.yaml
-   ```
- - configure the Active MQ self-signed certificates, the steps can be found [here](https://access.redhat.com/documentation/en-us/red_hat_amq/2020.q4/html/deploying_amq_broker_on_openshift/deploying_broker-on-ocp-using-templates_broker-ocp#connecting-external-clients-to-template-based-brokers_broker-ocp)
-
+ - install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
```
Comment (spolti) [inline quickstarts/hello-rules/README.md:91]:
```suggestion
 - install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
```

File: quickstarts/hello-rules/README.md [hunk 10]
```diff
@@ -125,35 +88,9 @@ by using the *hello-rules* quickstart and this client to interact with Active MQ
...
--p SOURCE_REPOSITORY_REF=main \
--p CONTEXT_DIR=quickstarts/hello-rules/hellorules \
--p AMQ_USERNAME=admin -p AMQ_PASSWORD=RedHat \
--p AMQ_SECRET=<amq_secret_with_trust_and_key_store> \
--p AMQ_TRUSTSTORE=<truststore_file_name> -p AMQ_TRUSTSTORE_PASSWORD=<truststore_password> \
--p AMQ_KEYSTORE=<keystore_file_name> -p AMQ_KEYSTORE_PASSWORD=<keystore_password> \
--p IMAGE_STREAM_NAMESPACE=openshift
-```
-
+After properly configuring the Active-MQ pre requisites, you have to wait that your KieApp will be ready and running before continuing.
```
Comment (spolti) [inline quickstarts/hello-rules/README.md:93]:
```suggestion
After properly configuring the Active-MQ pre requisites, you have to wait the KieApp gets ready before continuing.
```

File: quickstarts/library-process/README.md [hunk 11]
```diff
@@ -6,79 +6,41 @@ This quickstart is intend to be used with the [IBM BAMOE Kie Server](https://git
...
-$ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/main/ibm-bamoe8-image-streams..yaml -n openshift
+$ oc new-project ibm-bamoe-kieserver
+Now using project "ibm-bamoe-kieserver" on server "https://ocp-main.mycloud.com:8443".
 ```
 
 Note that, to pull the images the OpenShift must be able to pull images from registry.redhat.io, for more information
 please take a look [here](https://access.redhat.com/RegistryAuthentication)
 
+To create the IBM BAMOE environment on your OpenShift instance you need to install Business Automation Operator.
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```
Comment (spolti) [inline quickstarts/library-process/README.md:28]:
```suggestion
For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```

File: quickstarts/library-process/README.md [hunk 12]
```diff
@@ -6,79 +6,41 @@ This quickstart is intend to be used with the [IBM BAMOE Kie Server](https://git
...
 
-Deploy the `credentials secret` provided as example:
+Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```
Comment (spolti) [inline quickstarts/library-process/README.md:30]:
```suggestion
Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```

File: quickstarts/library-process/README.md [hunk 13]
```diff
@@ -129,43 +91,18 @@ by using the *library-process* quickstart and this client to interact with Activ
...
-   ```bash
-   $ oc replace -n openshift --force  -f \
-     https://raw.githubusercontent.com/jboss-container-images/jboss-amq-7-broker-openshift-image/78-7.8.0.GA/amq-broker-7-image-streams.yaml
-   ```
- - configure the Active MQ self-signed certificates, the steps can be found [here](https://access.redhat.com/documentation/en-us/red_hat_amq/2020.q4/html/deploying_amq_broker_on_openshift/deploying_broker-on-ocp-using-templates_broker-ocp#connecting-external-clients-to-template-based-brokers_broker-ocp)
-
-
-After properly configuring the Active-MQ pre-requisites, deploy the Kie Server using the AMQ Application template:
-To deploy the template execute the following command:
+- install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
```
Comment (spolti) [inline quickstarts/library-process/README.md:94]:
```suggestion
- install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
```

File: quickstarts/library-process/README.md [hunk 14]
```diff
@@ -129,43 +91,18 @@ by using the *library-process* quickstart and this client to interact with Activ
...
--p SOURCE_REPOSITORY_REF=main \
--p CONTEXT_DIR=quickstarts/library-process/library \
--p AMQ_USERNAME=admin -p AMQ_PASSWORD=RedHat \
--p AMQ_SECRET=<amq_secret_with_trust_and_key_store> \
--p AMQ_TRUSTSTORE=<truststore_file_name> -p AMQ_TRUSTSTORE_PASSWORD=<truststore_password> \
--p AMQ_KEYSTORE=<keystore_file_name> -p AMQ_KEYSTORE_PASSWORD=<keystore_password> \
--p IMAGE_STREAM_NAMESPACE=openshift
-```
 
+After properly configuring the Active-MQ pre-requisites, you have to wait that your KieApp will be ready and running before continuing.
```
Comment (spolti) [inline quickstarts/library-process/README.md:97]:
```suggestion
After properly configuring the Active-MQ pre-requisites, you have to wait the KieApp gets ready before continuing.
```

File: templates/contrib/jdbc/README.md [hunk 15]
```diff
@@ -174,43 +153,14 @@ After you build your extension image you can:
...
-```bash
-oc create -f extension-image-streams.yaml
-```
-
-At this point we are ready to create a Kie Server deployment using the extension driver:
-Note that, the driver name can be found in the respective install-{DB}. properties file, i.e. [mariadb](modules/kie-custom-jdbc-driver/added/install-mariadb.properties)
-
-To create a new app using a extension image, you can go through the OpenShift web console and fill all the needed fields or
-create it using command line:
+All the steps needed are documented [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.10/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator)
```
Comment (spolti) [inline templates/contrib/jdbc/README.md:158]:
```suggestion
All the steps needed are documented [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator) 
```

**Review by spolti — CHANGES_REQUESTED** [review]
Nice work Davide.
made some suggestions, and plus, make sure to remove the adocs generation command from the Makefile as well.

File: quickstarts/hello-rules-multi-module/README.md (hunk 2 above)
Comment (davidesalerno) [reply quickstarts/hello-rules-multi-module/README.md:28]: +1

**Comment by davidesalerno** [comment]
@spolti Thank you very much for your precious review.

I updated the links to the documentation referring to the existing ones of the 7.12 release and so using:
- https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-subscribe-proc_openshift-operator
- https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-environment-deploy-assy_openshift-operator
- https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-deploy-kieserver-proc_openshift-operator

and so in the code the links are referencing the 7.13 release.

Removed also the generate adocs target into the Makefile.

**Review by desmax74 — APPROVED** [review]

**Comment by spolti** [comment]
> @spolti Thank you very much for your precious review.
> 
> I updated the links to the documentation referring to the existing ones of the 7.12 release and so using:
> 
> * https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-subscribe-proc_openshift-operator
> * https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-environment-deploy-assy_openshift-operator
> * https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-deploy-kieserver-proc_openshift-operator
> 
> and so in the code the links are referencing the 7.13 release.
> 
> Removed also the generate adocs target into the Makefile.

My Pleasure man!

it is fine to keep as 7.12.

**Review by spolti — APPROVED** [review]

**Pull request merged**